server/services/store/storetests/boards.go

```diff
@@ -106,7 +105,7 @@ func testGetBoard(t *testing.T, store store.Store) {
 
 	t.Run("nonexisting board", func(t *testing.T) {
 		rBoard, err := store.GetBoard("nonexistent-id")
-		require.ErrorIs(t, err, sql.ErrNoRows)
+		require.True(t, model.IsErrNotFound(err))
```

Comment:
nit: A better error message would be great if not you get something like:

```
=== RUN   TestSQLStore/BoardStore/GetMemberForBoard/should_return_a_no_rows_error_for_nonexisting_membership
    boards.go:571: 
        	Error Trace:	boards.go:571
        	Error:      	Should be false
        	Test:       	TestSQLStore/BoardStore/GetMemberForBoard/should_return_a_no_rows_error_for_nonexisting_membership
```

Reply:
Added descriptions to the asserts.  https://github.com/mattermost/focalboard/pull/2834/commits/4fb77221e7f68c213be6de5865bb318dd20f2618